dubbo-common/src/main/java/org/apache/dubbo/config/ReferenceConfigBase.java

```diff
@@ -48,6 +48,8 @@
 
     private static final long serialVersionUID = -5864351140409987595L;
 
+    private static final String ORIGIN_CONFIG = "ORIGIN_CONFIG";
```

Comment:
You can put this `constant` in `CommonConstants` and modify `ServiceConfigBase` by the way